src/components/Form.tsx

```diff
@@ -1,98 +1,16 @@
 import React from 'react';
-import Button from '@material-ui/core/Button';
-import TextField from '@material-ui/core/TextField';
-import Grid from '@material-ui/core/Grid';
-import Box from '@material-ui/core/Box';
-import Typography from '@material-ui/core/Typography';
-import { makeStyles } from '@material-ui/core/styles';
-import Container from '@material-ui/core/Container';
+import NamePicker from './NamePicker';
+import LifeEventPicker from './LifeEventPicker';
+import RelationshipPicker from './RelationshipPicker';
+import CategoryPicker from './CategoryPicker';
 
 export default function SearchForm(): JSX.Element {
-  const classes = useStyles();
-
   return (
-    <Container component="main" maxWidth="xs">
-      <div className={classes.paper}>
-        <Typography component="h1" variant="h5">
-          Sign up
-        </Typography>
-        <form className={classes.form} noValidate>
-          <Grid container spacing={2}>
-            <Grid item xs={12} sm={6}>
-              <TextField
-                autoComplete="fname"
-                name="firstName"
-                variant="outlined"
-                required
-                fullWidth
-                id="firstName"
-                label="First Name"
-                autoFocus
-              />
-            </Grid>
-            <Grid item xs={12} sm={6}>
-              <TextField
-                variant="outlined"
-                required
-                fullWidth
-                id="lastName"
-                label="Last Name"
-                name="lastName"
-                autoComplete="lname"
-              />
-            </Grid>
-            <Grid item xs={12}>
-              <TextField
-                variant="outlined"
-                required
-                fullWidth
-                id="email"
-                label="Email Address"
-                name="email"
-                autoComplete="email"
-              />
-            </Grid>
-            <Grid item xs={12}>
-              <TextField
-                variant="outlined"
-                required
-                fullWidth
-                name="password"
-                label="Password"
-                type="password"
-                id="password"
-                autoComplete="current-password"
-              />
-            </Grid>
-          </Grid>
-          <Button
-            type="submit"
-            fullWidth
-            variant="contained"
-            color="secondary"
-            className={classes.submit}
-          >
-            Sign Up
-          </Button>
-        </form>
-      </div>
-      <Box mt={5}></Box>
-    </Container>
+    <div>
+      <NamePicker />
+      <LifeEventPicker />
+      <RelationshipPicker />
+      <CategoryPicker />
+    </div>
```

Comment:
I think it was a good move to break all those pickers out into their own components. It makes this top-level component so much simpler and lets you make each picker as simple or complex as it needs to be without making a mess of the code.

I do wonder, though, if the `<form>` element should wrap all these pickers. And maybe the outer Grid container? And the submit button?

Reply:
Noted, will look into the form <element> wrapping the pickers, outer grid container, and submit button!